backend/compact-connect/docs/design/README.md

```diff
@@ -85,16 +85,19 @@ how permissions are stored and translated into scopes.
 
 #### Compact Executive Directors and Staff
 
-Compact ED level staff can have permission to read all compact data as well as to create and manage users and their
-permissions. They can grant other users the ability to write data for a particular jurisdiction and to create more
-users associated with a particular jurisdiction. They can also delete any user within their compact, so long as that
-user does not have permissions associated with a different compact.
+Compact ED level staff can have permission to read all generally available data for any compact, and if they have the 
+`readPrivate` permission they will be able to view all data for any licensee within the compact. They will also be able
+to create and manage users and their permissions. They can grant other users the ability to write data for a particular
+jurisdiction and to create more users associated with a particular jurisdiction. They can also delete any user within 
+their compact, so long as that user does not have permissions associated with a different compact.
```

Comment:
Discussed with @jusdino, and it looks like the system currently does not even support deleting users, so I have removed that entire detail for now to avoid confusion.